interfaces/mount/spec.go

```diff
@@ -140,6 +140,15 @@ func (spec *Specification) AddLayout(si *snap.Info) {
 	}
 }
 
+// AddExtraLayouts adds mount entries based on additional layouts that
+// are provided for the snap.
+func (spec *Specification) AddExtraLayouts(ExtraLayouts []snap.Layout) {
```

Comment:
it's unusual to have uppercase arguments: s/ExtraLayouts/layouts/

Reply:
Ah good catch, I don't know how that one slipped in :-)